src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -149,18 +203,21 @@ const Conversation: React.FC<ConversationProps> = props => {
               {isOfferable && <BuyerGuaranteeMessage />}
               {inquiryItemBox}
               <Waypoint onEnter={loadMore} />
-              {fetchingMore ? (
-                <SpinnerContainer>
-                  <Spinner />
-                </SpinnerContainer>
-              ) : null}
+              {fetchingMore ? <Loading /> : null}
               <ConversationMessages
                 // @ts-expect-error STRICT_NULL_CHECK
                 messages={conversation.messagesConnection}
+                lastViewedMessageID={+(lastMessageID || -1)}
               />
               <Box ref={bottomOfPage}></Box>
             </Flex>
           </Box>
+          <UnreadMessagesToastQueryRenderer
+            conversationID={conversation?.internalID!}
+            onOfferable={!!(artwork && isOfferable)}
+            hasScrolled={!isBottomVisible}
+            onClick={viewUnreadMessages}
+          />
```

Comment:
This is the actual cta. `onOfferable` is used for positioning, but I'm sure there's a million smarter and less verbose ways of accomplishing the same.